pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -249,9 +289,12 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 
 	for rName, rQuant := range podRequest.ScalarResources {
 		if v1helper.IsExtendedResourceName(rName) {
-			// If this resource is one of the extended resources that should be
-			// ignored, we will skip checking it.
-			if ignoredExtendedResources.Has(string(rName)) {
+			// If this resource is one of the extended resources that should be ignored, we will skip checking it.
+			var rNamePrefix string
+			if ignoredResourceGroups.Len() > 0 {
```

Comment:
This conditional seems unnecessary.

However, if you have `example.com` in `ignoredResourceGroups`, but not in `ignoredResources`, the resource `example.com` would be ignored. Is this what you want?

This needs to be very clear in the godoc for the type.

Reply:
the resource `example.com` is not invalid since it's not an extended resource.

Reply:
add a comment saying that `rName` is guaranteed to have a slash due to API validation.